org.ryujinx.Ryujinx.json

```diff
@@ -0,0 +1,1291 @@
+{
+    "app-id": "org.ryujinx.Ryujinx",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "19.08",
+    "sdk": "org.freedesktop.Sdk",
+    "sdk-extensions": ["org.freedesktop.Sdk.Extension.dotnet"],
+    "command": "start",
+    "finish-args" : [
+        "--share=ipc",
+        "--share=network",
+        "--device=dri",
+        "--socket=x11",
+        "--filesystem=host"
```

Comment:
If configuration follows XDG spec then `--filesystem=home` and `--filesystem=/run/media' would be still better.